x/docs/md/intro/getting-started.md

```diff
@@ -14,3 +14,11 @@ Once you have submitted your first exercise, there are several other things you
 1. Browse other solutions to the same problem.
    Provide feedback: ask questions, suggest improvements, and ponder style and readability.
 1. Fetch a new exercise and solve it.
+
+## Command Line Tips 
+
+You can submit a header file along with your implementation using:  
```

Comment:
Would it be correct to say that I could also submit multiple files 
cd path/to/folder
exercism submit file1 file2